src/app/pages/bajas/form-solicitud/form-solicitud.component.ts

```diff
@@ -26,14 +29,13 @@ export class FormSolicitudComponent implements OnInit {
   tiposBaja: FormatoTipoMovimiento[];
   sizeSoporte: number = 5;
```

Comment:
Dado que ya se está haciendo en el `constructor`
```suggestion
  sizeSoporte: number;
```